spec/features/rooms/index_spec.rb

```diff
@@ -0,0 +1,22 @@
+require "rails_helper"
+
+RSpec.describe "Rooms Index", type: :feature do
+  before (:each) do
+    @hotel_1 = Hotel.create!(name: "Echo Mountain Inn", location: "Echo Mountain")
+    @room_1 = Room.create!(rate: 125, suite: "Presidential", hotel: @hotel_1)
+    @guest_1 = Guest.create!(name: "Charlize Theron", nights: 3)
+    @guest_1_room_1 = GuestRoom.create!(guest_id: @guest_1.id, room_id: @room_1.id)
+  end
+  describe "As a visitor" do
+    describe "When I visit the rooms index page" do
+      it "Then I see a list of all rooms including the room's suite, nightly rate, and the name of the hotel that it belongs to and the number of guests that have stayed in that room." do
+        visit "/rooms"
+        save_and_open_page 
```

Comment:
when you're pushing up code, take these methods out so it doesn't affect when other devs pull down your code and run your test suite.